cf/api/resources/service_keys.go

```diff
@@ -0,0 +1,37 @@
+package resources
+
+import "github.com/cloudfoundry/cli/cf/models"
+
+type ServiceKeyResource struct {
+	Resource
+	Entity ServiceKeyEntity
+}
+
+type ServiceKeyEntity struct {
+	Name                string                 `json:"name"`
+	ServiceInstanceGuid string                 `json:"service_instance_guid"`
+	ServiceInstanceUrl  string                 `json:"service_instance_url"`
+	Credentials         map[string]interface{} `json:"credentials"`
+}
+
+func (resource ServiceKeyResource) ToFields() (fields models.ServiceKeyFields) {
```

Comment:
the returned var `fields` is not needed. The func declaration could just read
`func (resource ServiceKeyResource) ToFields() models.ServiceKeyFields`